### PR TITLE
Add Japanese base and multiple preset base support

### PR DESCRIPTION
**Pull request by yvbbrjdr**
### Submitter Checklist

- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Tested on Firefox (`npm run firefox`)
- [x] Tested on Chromium (`npm run chromium`)

### CODE REVIEW

File: src/options/options.ts [hunk 1]
```diff
@@ -176,6 +176,17 @@ $( document ).ready( async () => {
   $( '#passphraseBox' ).keydown( entryModalOnEnter );
   $( '#baseBox' ).keydown( entryModalOnEnter );
   $( '#tagsBox' ).keydown( entryModalOnEnter );
+
+  const presetDropdownItems = $( '#presetDropdownItems' );
```
Comment (keur) [inline src/options/options.ts]: You can type this as an `HTMLDivElement`

File: src/utils/cryptoutils.ts [hunk 2]
```diff
@@ -22,8 +22,17 @@
 import * as _sodium from 'libsodium-wrappers';
 const babbleSalt: string = 'BabbleBabbleBabb';
 
-export const babbleDefaultBase: string =
-  '的一是了人在有我他这为之来以个们到说和地也子时而要于就下得可你年生自会那后能对着事其里所去行过十用发如然方成者多都三二同么当起与好看学进将还分此心前面又定见只从现因开些长明样已月正想实把相两力等外它并间手应全门点身由何向至物被五及入先己或很最书美山什名曰合加世水果位度马给数次今表原各才几解则气再听提万更比百尔即白许系且光路接结题指感量取场电空边件住放风求形望传笑叫往区达设记字故品象花七服据云像飞远收石类程转千式流每该始术格运怎步让识拉若备造快集布尽称确呢节注存具甚容吃算坐早引似视尚轻况留照写足余星居技属找食';
+export const babbleDefaultBases: string[][] =
```
Comment (keur) [inline src/utils/cryptoutils.ts]: Make this a `dict`, or even a `Base` type

File: src/options/options.ts (hunk 1 above)
Comment (yvbbrjdr) [reply src/options/options.ts]: I don't know if `HTMLDivElement` will work with jQuery `.click()`.

Comment (keur) [reply src/options/options.ts]: fucking jquery...

**Comment by yvbbrjdr** [comment]
@keur It's not possible to "decompose hangul characters into" Korean letters since there are only 24 letters.

**Pull request merged**